pycbc/strain.py

```diff
@@ -377,8 +377,8 @@ def from_cli(opt, dyn_range_fac=1, precision='single',
                                           low_frequency_cutoff=lowfreq)
             strain = resample_to_delta_t(strain, 1.0/opt.sample_rate)
 
-        if opt.injection_file or opt.sgburst_injection_file \
-                or opt.ringdown_injection_file and not opt.channel_name:
+        if not opt.channel_name and opt.injection_file \
+                or opt.sgburst_injection_file or opt.ringdown_injection_file:
```

Comment:
Should there be parenthesis around the injection file options? i.e. 

not opt.channel_name and (opt.injection_file or opt.sgburst_injection file....)

Reply:
I just tested it providing a ringdown-injection-file and not a channel-name, as well as neither a channel-name nor any kind of injection file, and it seems to be working. But I can add a parenthesis if you want

Reply:
Actually, with the parenthesis I don't think it works

Reply:
Try it if you *have* give a channel name *and* a ringdown injection file ;-)

Reply:
Good point. I thought I had tried all possible combinations but I probably got confused with what I tried before making the changes and what I had tried after! The parenthesis is indeed needed, I'll push it immediately :)